### PR TITLE
Probably fix Uncaught TypeError in ripple.ts

### PR DESCRIPTION
**Pull request by DarthGandalf**
See also: #6477

## Description

I randomly got an error `ripple.js:109 Uncaught TypeError: Cannot read property 'touched' of undefined` (the line numbers are off due to my build system, I think)

## Motivation and Context
To not crash.

## How Has This Been Tested?
I don't know how to reproduce the issue. But this look like the only line in ripple.ts which assumes that `_ripple` exists before accessing its `touched`.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/directives/ripple.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ function isRippleEnabled (value: any): value is true {
 function rippleShow (e: MouseEvent | TouchEvent) {
   const value: RippleOptions = {}
   const element = e.currentTarget as HTMLElement
-  if (!element || element._ripple!.touched) return
+  if (!element || !element._ripple || element._ripple.touched) return
   if (isTouchEvent(e)) {
     element._ripple!.touched = true
```
Comment (KaelWD) [inline packages/vuetify/src/directives/ripple.ts]: Remove all the non-null assertions.

Comment (DarthGandalf) [reply packages/vuetify/src/directives/ripple.ts]: Done

**Comment by Himself65** [comment]
can add one more unit tests？

**Comment by DarthGandalf** [comment]
@Himself65 I don't know how to test it. In packages/vuetify/test/unit/directives/ripple.spec.js properties of `_ripple` are already accessed without triggering this.

**Pull request merged**